docs/source/dev_guide/algorithms/decision_forest/decision-forest.rst

```diff
@@ -49,10 +49,22 @@ internal nodes :math:`t` represent a binary test (split) dividing their
 subset :math:`X_t` in two subsets :math:`X_{t_L}` and :math:`X_{t_R}`,
 corresponding to their children :math:`t_L` and :math:`t_R`.
 
+.. _df_inexact_hist_method:
+
+Inexact Histogram Computation Method
+++++++++++++++++++++++++++++++++++++
+
+In inexact histogram method only a selected subset of splits is considered for computation of a best split.
```

Comment:
Can we add reference to bibliography to there details (how subset is specified and so on) can be found?

Reply:
There is no article the whole algorithm is based on exactly.

Reply:
@makart19 is there a reference we can add?

Reply:
no input